conf/scalar-loki-stack-custom-values.yaml

```diff
@@ -0,0 +1,69 @@
+promtail:
+
+  nodeSelector:
+    scalar-labs.com/dedicated-node: scalardb
+    # scalar-labs.com/dedicated-node: scalardl-ledger
+    # scalar-labs.com/dedicated-node: scalardl-auditor
+
+  tolerations:
+    - effect: NoSchedule
+      key: scalar-labs.com/dedicated-node
+      operator: Equal
+      value: scalardb
+      # value: scalardl-ledger
+      # value: scalardl-auditor
```

Comment:
In the production environment, it is recommended to set taints/tolerations to make specific worker nodes dedicated to Scalar products.
https://github.com/scalar-labs/scalar-kubernetes/blob/reorganize-docs/docs/CreateAKSClusterForScalarProducts.md#add-taint-to-the-worker-node-that-is-used-for-toleration

Since the promtail pods are deployed as DaemonSet, we need to add tolerations in the promtail configuration.
In other words, if we don't set tolerations in the custom values file of promtail, the promtail pods cannot be deployed on the worker node dedicated to Scalar products.

So, I added the `tolerations` setting in the sample custom values file.
With this tolerations configuration, the promtail pods can be deployed on the worker node dedicated to Scalar products.